### PR TITLE
Forget pwd changes

### PR DESCRIPTION
**Pull request by ElakkiyaMohanRajan**
#### What's this PR do? Any additional context?
Changing the forget pwd change text
#### Where should the reviewer start?
Click on the forget password button and enter the email . Now you can see the text changes and show the email address to where the password reset email has been sent
#### How can this be manually tested? (brief repro steps)
The text was not clear and there was no indication of email to where the password resent email has been sent
#### What are the relevant TFS items? (list id numbers)
88245
#### Definition of done:
- [ ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [x] For large or complex change: schedule an in-person review session

[//]: # ( todo: Is there appropriate test coverage? )
[//]: # ( todo: Does this PR require a new Selenium test? )
[//]: # ( todo: Is there appropriate logging/monitoring included? )

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging into the development branch then "flatten" or "squash" commits
1. If merging from development into master then don't "flatten" or "squash" commits

#### Reminders AFTER merging
1. Delete the remote branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc)

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 3 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 4 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 5 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

**Comment by ElakkiyaMohanRajan** [comment]
Need the additional theming change . So created the separate pull request https://github.com/Microsoft/edx-theme/pull/86

File: common/djangoapps/student/views.py [hunk 1]
```diff
@@ -2220,10 +2220,10 @@ def password_reset(request):
         # bad user? tick the rate limiter counter
         AUDIT_LOG.info("Bad password_reset user passed in.")
         limiter.tick_bad_request_counter(request)
-
+    from_email=request.POST.get('email')
```
Comment (sdolenc) [inline common/djangoapps/student/views.py]: nit: please put spaces on either side of equal operator before merging

Comment (sdolenc) [inline common/djangoapps/student/views.py]: fyi, there are great "linter" tools for python that'll help automate this for you too. just google "vs code python plugin" or "vs code python linter" . i'm sure there are settings for pycharm too

**Comment by manikarthikk** [comment]
Change looks good.
Please Run the test cases related to this change in view file and fix them if needed.

**Comment by ElakkiyaMohanRajan** [comment]
Included the space in the from_email line

**Pull request merged**